### PR TITLE
Combustion fixes jeos

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [combustion test suite failures](https://progress.opensuse.org/issues/154093)

#### Verification runs: 

* [jeos-15sp6](http://kepler.suse.cz/tests/22599#)
* [12sp5](http://kepler.suse.cz/tests/22594)

### CODE REVIEW

File: tests/console/verify_efi_mok.pm [hunk 1]
```diff
@@ -140,24 +141,23 @@ sub get_esp_info {
         svirt_hyperv => 'scsi'
     };
     # return a the first element (drive or partition number) from parted's output
```
Comment (Vogtinator) [inline tests/console/verify_efi_mok.pm:143]: Comment doesn't match the code

Comment (mloviska) [reply tests/console/verify_efi_mok.pm]: thanks, that was a left over including the above hash

**Review by clanig — APPROVED** [review]
LGTM

File: tests/jeos/grub2_gfxmode.pm [hunk 2]
```diff
@@ -12,13 +12,16 @@
...
-use serial_terminal 'select_serial_terminal';
+use serial_terminal;
 use jeos qw(set_grub_gfxmode);
+use utils qw(ensure_serialdev_permissions);
 
 sub run {
     select_serial_terminal;
 
     set_grub_gfxmode;
+    ensure_serialdev_permissions;
```
Comment (Vogtinator) [inline tests/jeos/grub2_gfxmode.pm:23]: This is a bit out of place here IMO

Does this work in all scenarios?

Comment (mloviska) [reply tests/jeos/grub2_gfxmode.pm:23]: It has worked here http://kepler.suse.cz/tests/22594#step/grub2_gfxmode/32 despite that it was done in first_run test module as well. I have rather put this in a test module than into combustion

Comment (Vogtinator) [reply tests/jeos/grub2_gfxmode.pm:23]: If firstrun already performs `ensure_serialdev_permissions` it shouldn't be necessary to do it again. Maybe the `dialout` group is missing or something else breaks it?

Comment (Vogtinator) [reply tests/jeos/grub2_gfxmode.pm:23]: Right, now I get it. Arguably whatever creates the user is responsible for calling `ensure_serialdev_permissions` and grub2_gfxmode is not the right place. If doing it in combustion is not an option, `verify_setup` or `system_prepare` might fit

Comment (mloviska) [reply tests/jeos/grub2_gfxmode.pm:23]: That is true, I want to turn `grub2_gfxmode` module into something like https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/host_config.pm but for minimal-vm hosts. I haven't just renamed the module in this PR

Comment (mloviska) [reply tests/jeos/grub2_gfxmode.pm:23]: And verification module should not do system-wide changes IMHO

**Review by ricardobranco777 — APPROVED** [review]

**Pull request merged**